app/Http/Controllers/DeleteAirlineController.php

```diff
@@ -3,14 +3,14 @@
 namespace App\Http\Controllers;
 
 use App\Models\Airline;
-use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
 
 class DeleteAirlineController extends Controller
 {
-    public function __invoke(Airline $airline): JsonResponse
+    public function __invoke(Airline $airline): Response
```

Comment:
thats because when I put a JsonResponse I have to transform the request to json like this 
<img width="313" alt="image" src="https://github.com/LukiMaurojorge/flight-agency/assets/141275702/ed7b223d-92a9-48da-bf84-365e36bdda79">

Reply:
I guess in this case it doesn't matter because it's an empty response.